multinet/uploaders/swagger/csv.yaml

```diff
@@ -16,6 +17,13 @@ parameters:
         0,picard,captain
         1,riker,commander
         2,data,lieutenant commander
+  -
+    name: key
+    in: query
```

Comment:
Is this shorthand for query string? Maybe it would be useful to new users to specify the whole thing?

Reply:
This just specifies to the api docs that it's a query param, not a body or path param.